### DISCUSSION TITLE
Should there be a note in README about the project going on infinite hiatus?

### DISCUSSION THREAD

**Discussion by 88572** [virtual-puppet-project/vpuppr#221] [General]
the title

**Comment by NickOveracker** [comment]
The commit activity doesn't really look like it's on hiatus to me. It's just managed by a busy individual rather than some well-funded NPO.

**Reply by ItsRogueRen** [reply]
Youwin just announced on Discord a few days ago the project is going into an indefinite hiatus. An update to the readme may come at some point.

**Reply by VoxelLibrary** [reply]
I can corroborate what Ren said.

**Reply by CartoonFan** [reply]
> Youwin just announced on Discord a few days ago the project is going into an indefinite hiatus. An update to the readme may come at some point.

The Discord invite link doesn't work for me (and the server doesn't appear in my list). Is there anywhere else I can see this notice?

**Comment by omega7379** [comment]
additionally, should I add a note to the [website](https://vpup.pro/) or will that also be taken down?

**Comment by 88572** [comment]
I would recommend leaving the website up for now with a note about the
situation.

**Reply by omega7379** [reply]
there's a [PR](https://github.com/virtual-puppet-project/website/pull/10) open to change the website open

**Comment by nova-37** [comment]
Signed into my old account to thank you for this note. I had gotten really frustrated with the software and had no clue it'd been on hiatus.
If it weren't for this Github post, I wouldn't know.
Hope there's an alternative out there.
-Nova
